FirebaseTestProject/Views/PostRow.swift

```diff
@@ -8,25 +8,92 @@
 import SwiftUI
 
 struct PostRow: View {
-    let post: Post
+    @Binding  var post: Post
+    @State var isPresenting: Bool = false
+    @State private var showAlert: Bool = false
+    @State private var systemImage: String = ""
```

Comment:
The `systemImage` variable is not used. We can probably remove it.

Reply:
Whoops - old version of my codebase! :) Thanks!